packages/next-codemod/bin/transform.ts

```diff
@@ -130,6 +130,7 @@ export async function runTransform(
     args.push('--verbose=2')
   }
   args.push('--no-babel')
+  args.push('--parser=tsx')
```

Comment:
The TypeScript files I tested didn't parse correctly without this. If the user wants a specific parser, I think they can still use the `--jscodeshift` option to specify args directly and override it